lib/drivers/veres/index.js

```diff
@@ -409,7 +409,7 @@ api.rotate = async options => {
   logger._log(options, 'Saved updated document.');
 
   // Update was successful, save new private key to key store
-  await keyStore().put(did, await didDocument.exportKeys());
+  await keyStore().put(did, _exportPrivateKeys({didDocument}));
```

Comment:
This is passing `didDocument` but `_exportPrivateKeys` takes `keyPairs`. Not sure what's going on there.